docker/base.Dockerfile

```diff
@@ -20,14 +20,19 @@ RUN apt-get -qq update && apt-get -qq install -y \
     libz-dev \
     locales \
     perl \
-    postgresql \
+    postgresql-common \
     rsync \
     sqlite3 \
     wget \
     libicu-dev \
     pkg-config \
     && rm -rf /var/lib/apt/lists/*
 
+ARG POSTGRESQL_VERSION=13
+RUN /usr/share/postgresql-common/pgdg/apt.postgresql.org.sh -y \
+    && apt -qq install -y postgresql-${POSTGRESQL_VERSION} \
+    && rm -rf /var/lib/apt/lists/*
```

Comment:
> Instead of installing PostgreSQL from Debian repositories, use the PostgreSQL Apt Repository to install its lowest-supported version.

Do we do this anywhere else? I'd be more keen if this magic was already accepted elsewhere

Reply:
4e15850c for repo usage

0c73ed7b for min version